lib/features/rectangles/presentation/pages/Rectangle.dart

```diff
@@ -0,0 +1,199 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+// ignore: file_names
+
+import 'dart:io';
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+import 'package:the_boxesv3/features/rectangles/domain/entities/rectiangle_image.dart';
+import 'package:the_boxesv3/features/rectangles/presentation/bloc/rectangle_imports.dart';
+import 'package:the_boxesv3/injection_container.dart';
+
+class Rect42 extends StatelessWidget {
+  final int count;
+
+  const Rect42({Key? key, required this.count}) : super(key: key);
+
+  int getCount() {
+    return count;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Rectangle Image List'),
+      ),
+      body: buildBody(),
+    );
+  }
+
+  BlocProvider<RectangleBloc> buildBody() {
+    sl.allowReassignment = true;
+    sl.registerLazySingleton(() => getCount());

```

Comment:
not understanding the point for that

Reply:
I do this to register a lazy singleton for the getCount function. I wasn't able to do this in the injection container because it's a static function and I needed an instance of Rect42 in order to call it.